lib/vanagon/platform/windows.rb

```diff
@@ -6,7 +6,7 @@ class Windows < Vanagon::Platform
       # @param project [Vanagon::Project] project to build a windows package of
       # @return [Array] list of commands required to build a windows package for the given project from a tarball
       def generate_package(project)
-        case project.platform.package_type
+        case @package_type
```

Comment:
If we are changing this, we should also change Line 24 below - otherwise leave this as-is